Source/ThirdParty/ANGLE/ANGLE.xcodeproj/project.pbxproj

```diff
@@ -3985,6 +5246,753 @@
 			};
 			name = Production;
 		};
+		7BFBD7AF2C7DE55300BFCA94 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = 7BFBD7E02C7DE7C700BFCA94 /* ANGLETranslatorFuzzerStaticAssert.xcconfig */;
+			buildSettings = {
+				ALWAYS_SEARCH_USER_PATHS = NO;
```

Comment:
Could you please remove the default build settings in all these new XCBuildConfiguration objects? They should be unnecessary to build, as long as each target has a `Base.xcconfig` in its includes and defines `PRODUCT_NAME` in xcconfigs.

Reply:
Oops, I also mentioned this in my review (which I started before lunch, then just finished).  It's the same as all of the "Ditto" comments, plus the original comment that the Dittos referred to.